code/src/components/Details.css

```diff
@@ -0,0 +1,113 @@
+.detailsPage {
```

Comment:
I get really inspired by your styling in general. Will definitely try to implement some similar code in my future projects. :)

Reply:
Thank you Rebecca!